test/functional/flutter_integration/commands_test.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from appium.webdriver.common.appiumby import AppiumBy
+from appium.webdriver.extensions.flutter_integration.scroll_directions import ScrollDirection
+from appium.webdriver.flutter_finder import FlutterFinder
+from test.functional.flutter_integration.helper.test_helper import BaseTestCase
+
+class TestFlutterCommands(BaseTestCase):
+    
+    def test_wait_command(self):
+        self.__open_screen('Lazy Loading')
+    
+        message_field_finder = FlutterFinder.by_flutter_key('message_field')
```

Comment:
not sure if we need to repeat `flutter` twice. Maybe rename `FlutterFinder.by_flutter_key` to `FlutterFinder.by_key`?

Reply:
Agreed! Updated.